src/navigator/IFrameNavigator.ts

```diff
@@ -426,7 +426,7 @@ export default class IFrameNavigator implements Navigator {
     this.headerMenu = headerMenu;
     this.mainElement = mainElement;
     try {
-      var wrapper = HTMLUtilities.findRequiredElement(
+      let wrapper = HTMLUtilities.findRequiredElement(
```

Comment:
Should this be a `const` if we are not reassigning it?

Reply:
I guess it could , I am sure there are plenty of other places where that needs to be evaluated again too. This one was just a result of removing the re-declaration of it.

Reply:
Ok yep we can address that later